### ISSUE TITLE
Fine-tune DeepLabv3 (pre-trained on BDD100K) over Cityscape

### ISSUE THREAD

**Issue by kartikgupta2607** [open-mmlab/mmsegmentation#1662]
I want to Fine-tune DeepLabv3 (pre-trained on BDD100K) over the Cityscape dataset and keep the output labels the same as 19. Can someone explain how to freeze layers in mmseg DeepLabv3 model and train the remaining ones, as I am thinking of training only the last few layers? I am also open to any other approach for fine-tuning.

**Comment by linfangjian01** [comment]
Please ref https://github.com/open-mmlab/mmsegmentation/blob/f3c72c6dbb9b5d9f382a06d1bdf6d59f1a83e6ee/mmseg/models/backbones/vit.py#L435-L440. You can freeze some layers in this way.

**Comment by linfangjian01** [comment]
like `for param in model.backbone.parameters():
 param.requires_grad = False`.

**Comment by linfangjian01** [comment]
if freeze bn, dropout layer,  ref https://github.com/open-mmlab/mmsegmentation/blob/2dede047032c592285667225cb9ce67dc5b21c79/mmseg/models/backbones/vit.py#L435-L440 , others can replace `m.eval()` with `m.requires_grad_(False)`.
